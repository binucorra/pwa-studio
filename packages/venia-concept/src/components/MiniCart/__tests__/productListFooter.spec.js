import React from 'react';
import { createTestInstance } from '@magento/peregrine';

import ProductListFooter from '../productListFooter';

jest.mock('../totalsSummary');
jest.mock('src/components/Checkout', () => {
    return jest.fn().mockReturnValue('( Checkout Component Here )');
});
jest.mock('src/components/Checkout/checkoutButton', () => {
    return jest.fn().mockReturnValue('( Checkout Button Component Here )');
});

test('renders totals summary and checkout components', () => {
    const props = {};
    const tree = createTestInstance(<ProductListFooter {...props} />).toJSON();

    expect(tree).toMatchSnapshot();
});
