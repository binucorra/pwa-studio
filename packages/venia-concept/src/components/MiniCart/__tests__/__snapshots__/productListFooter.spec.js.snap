// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders totals summary and checkout components 1`] = `
<div>
  ( Totals Summary Component Here )
  ( Checkout Component Here )
</div>
`;
